releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -1,12 +1,34 @@
+# spring-cloud-azure-starter-appconfiguration-config
+
+[Source code][source_code_starter] | [Package (Maven)][package_starter] | [Product documentation][docs] | [Samples][samples]
+
+[CHANGELOG](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/spring/spring-cloud-azure-starter-appconfiguration-config/CHANGELOG.md)
+
 # spring-cloud-azure-appconfiguration-config
 
-[Source code][source_code] | [Package (Maven)][package] | [Product documentation][docs] | [Samples][samples]
+[Source code][source_code] | [Package (Maven)][package]
+
+[CHANGELOG](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/spring/spring-cloud-azure-appconfiguration-config/CHANGELOG.md)
 
 # spring-cloud-azure-appconfiguration-config-web
 
-[Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
+[Source code][source_code_web] | [Package (Maven)][package_web]
+
+## 4.7.0 - April 06, 2023
+
+### Features Added
 
-[Latest Library](./AzureSpringCloudAppConfigurationConfig)
+* Added support for [Azure Spring common configuration properties][azure_spring_common_configuration_properties].
+* Added support for Geo-Replication, enables the library to connect to a replica when the origin store fails to connect. Configuration is done through the `spring.cloud.azure.appconfiguration.stores[0].endpoints` property or `spring.cloud.azure.appconfiguration.stores[0].connection-strings`, where a list of endpoints/connection-strings can be provided. The endpoints/connection-strings will be attempted to connect in priority order.
```

Comment:
It's not necessary about the origin at all.
 
> Added support for Geo-Replication, enables the library to connect to a replica when the origin store fails to connect.

Added failover support for App Configuration stores with geo-replication enabled.